conversions/src/Data/Conversions/Hedgehog.hs

```diff
@@ -0,0 +1,110 @@
+module Data.Conversions.Hedgehog
+  ( BiConvert(..)
+  , Generators(..)
+  , mkConvertEither
+  , run
```

Comment:
Comment: I plan to use `Data.Conversions.Hedgehog` outside current `mhs/conversions` tests

Reply:
gotcha if we are re-exporting disregard my comment about reducing library dependencies.